docs/ref/middleware.txt

```diff
@@ -181,8 +181,6 @@ header, the middleware adds one if needed. If the response has a ``ETag`` or
 ``If-Modified-Since``, the response is replaced by an
 :class:`~django.http.HttpResponseNotModified`.
 
-Also sets ``Content-Length`` response-header.
-
```

Comment:
It might be worth changing this to a `versionchanged:: 1.11` by mentioning it was removed.